crates/egui_plot/src/lib.rs

```diff
@@ -975,10 +975,10 @@ impl<'a> Plot<'a> {
         }
 
         // Reset bounds to initial bounds if they haven't been modified.
-        if mem.auto_bounds.x {
+        if !default_auto_bounds.x || mem.auto_bounds.x {
             bounds.set_x(&min_auto_bounds);
```

Comment:
This reads backwards to me now. If the user has said `DON'T apply automatic bounds` then we set the bounds to `min_auto_bounds`

Reply:
The min_auto_bounds includes the bounds manually specified by include_x()/include_y(). So if you turn off auto bounds, and supply bounds via include_xy(), it won't use them except for the first call.